target/start-mailserver.sh

```diff
@@ -95,7 +97,7 @@ function register_functions() {
 
 	################### >> fix funcs
 
-	_register_fix_function "_fix_var_mail_permissions"
+#	_register_fix_function "_fix_var_mail_permissions"
```

Comment:
* all dovecot configuration is based on `docker:docker`(`5000:5000`). This should be kept.
* if this function becomes useless, it should be remove and not only commented out.